src/Evolution/Systems/ForceFree/TimeDerivativeTerms.hpp

```diff
@@ -0,0 +1,119 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Evolution/Systems/ForceFree/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace ForceFree {
+
+/*!
+ * \brief Compute the time derivative of the conserved variables for the GRFFE
+ * equations with divergence cleaning.
+ *
+ */
```

Comment:
Did I miss somewhere that this was documented? We usually give the equations explicitly and add a reference. Note that there are several FFE formulations, so the reference is important.

Reply:
OK, I see this is all in the other PR.

Reply:
The documentation has been spread between several structs in other systems (fluxes on the fluxes struct, sources on the sources struct, etc.) which has made it difficult to see the full evolution equations at once. Yoonsoo's seeing how documenting them on the system struct (see this PR: https://github.com/sxs-collaboration/spectre/pull/4440 ) works out. With subcell there's also a second `TimeDerivative` class and so I think moving the full documentation of the system to either the `System` class, or maybe actually the namespace (e.g. `GeneralizedHarmonic` or `ForceFree`) will make it easier to find and also easier to copy the equations into papers. Do you have any thoughts on this change/direction?

Reply:
I don't feel strongly about this - perhaps it is better to have all the documentation in one place. I only commented because I was expecting to see documentation here based on the other systems.

Reply:
Ah okay. Sounds good!